server/routes/creatingLicences/handlers/additionalPssConditionInput.ts

```diff
@@ -30,4 +31,27 @@ export default class AdditionalPssConditionInputRoutes {
       }`
     )
   }
+
+  DELETE = async (req: Request, res: Response): Promise<void> => {
+    const { licence } = res.locals
+    const { conditionId } = req.params
+    const { username } = req.user
+
+    const additionalConditionCodes = licence.additionalPssConditions
+      .filter((condition: AdditionalCondition) => condition.id !== parseInt(conditionId, 10))
+      .map((condition: AdditionalCondition) => condition.code)
+
+    await this.licenceService.updateAdditionalConditions(
+      licence.id,
+      LicenceType.PSS,
+      { additionalConditions: additionalConditionCodes },
```

Comment:
Wondered if we should have a specific enum for `ConditionType.Additional` and `ConditionType.PSS` in place of overloading the use of LicenceType.AP | .PSS ? This did catch me out for little while on a previous change until I realised.

Reply:
Yes we should, agree